src/hub_predtimechart/schema.py

```diff
@@ -0,0 +1,57 @@
+ptc_config_schema = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/reichlab/hub-dashboard-predtimechart/src/hub_predtimechart/schema.json",
+    "title": "predtimechart options",
+    "description": "hub predtimechart configuration object schema",
+    "type": "object",
+    "additionalProperties": False,
+    "properties": {
+        "rounds_idx": {
+            "description": "0-based index of the `rounds` entry to use for visualization",
+            "type": "integer",
+            "minimum": 0
+        },
+        "model_tasks_idx": {
+            "description": "0-based index of the `model_tasks` entry under `rounds_idx` to use for visualization",
+            "type": "integer",
+            "minimum": 0
+        },
+        "reference_date_col_name": {
+            "description": "name of the column that represents the `reference_date`",
```

Comment:
I'll keep beating this dead horse, but `reference_date` here means `round_id` doesn't it?

As written, this makes it sound like ptc can only be used for hubs that use a field called `reference_date` as their round_id.

💀 🐴

Reply:
@elray1 : Becky has an opinion about naming of this column. Let's talk about it soon.